lib/server/web.js

```diff
@@ -39,8 +39,18 @@ function trimLocale(header) {
 // This is the webserver. It's what the outside always talks to. It
 // handles the whole Profile API.
 exports.create = function createServer() {
+  var useRedis = config.serverCache.useRedis;
+  var cache = {
+    engine: useRedis ? require('catbox-redis') : require('catbox-memory')
+  };
+  if (useRedis) {
+    cache.host = config.serverCache.redis.host;
+    cache.port = config.serverCache.redis.port;
+    cache.partition = config.serverCache.redis.partition;
+  }
   var isProd = config.env === 'production';
   var server = new Hapi.Server({
+    cache: cache,
```

Comment:
Ah okay, that's fine. I was more worried about the engine. Looking at https://github.com/hapijs/hapi/blob/279def5a6b4246a7d9e6e86cd4ba2159475e1aec/lib/server.js#L70 we should be fine